idShiftUtils/go.mod

```diff
@@ -0,0 +1,3 @@
+module github.com/nestybox/sysbox/idShiftUtils
+
+go 1.14
```

Comment:
This reminds me that we should do this in all modules: s / 1.1? / 1.16

We 'own' the build-infra so it's up to us to define the compilation requirements.

Reply:
Agreed. Done for idShiftUtils.